stl/inc/chrono

```diff
@@ -5944,6 +5952,26 @@ namespace chrono {
             const auto _Month        = _Time.tm_mon + 1;
             const bool _Has_modifier = _Spec._Modifier != '\0';
             switch (_Spec._Type) {
+            case 'a':
+            case 'A':
+            case 'u':
+            case 'w':
+                if constexpr (_Is_any_of_v<_Ty, year_month_weekday, year_month_weekday_last>) {
+                    if (!_Val.weekday().ok()) {
+                        _THROW(format_error("Cannot print invalid weekday"));
+                    }
+                    _CharT _Fmt[3];
+                    _Fmt[0] = '%';
+                    _Fmt[1] = static_cast<_CharT>(_Spec._Type);
+                    _Fmt[2] = '\0';
+                    _Os << _STD put_time(&_Time, _Fmt);
+                    return true;

```

Comment:
I know that `put_time` doesn't have support for the `O` modifiers right now, but in `_Write` we still pass along the specifiers so when that magical day comes when `put_time` can handle them, everything should theoretically just work...should we also include the potential `O` modifier (valid for `%u` and `%w`)?

Reply:
Good call! I forgot that `u` and `w` support `O`. I'll add support.